float32  Time
float32 AccelX
float32 AccelY
float32 AccelZ

float32 GyroX
float32 GyroY
float32 GyroZ

float32 MagX
float32 MagY
float32 MagZ

float32 q0
float32 q1
float32 q2
float32 q3
